src/assets/locales/fr.json

```diff
@@ -0,0 +1,567 @@
+  "rtc-debug-panel.stats-panel-title": "Stats",
+  "rtc-debug-panel.track-panel-title": "Track",
+  "safari-mic-modal.message": "<p>Hubs nécessite les permissions du microphone dans Safari.</p><p>S'il vous plait, rechargez et autorisez l'accès au microphone pour continuer.</p>",
+  "safari-mic-modal.reload-button": "Recharger",
+  "safari-mic-modal.title": "Accès au microphone requis",
+  "scene-page.attribution": "<a>{name} par {author}</a>",
+  "scene-page.attribution-with-source": "<a>{name} par {author} sur {source}</a>",
+  "scene-page.create-button": "Créer une salle avec cette scène",
+  "scene-page.default-tweet": "{sceneName} dans {shareHashtag}",
+  "scene-page.edit-button": "Editer dans {editorName}",
+  "scene-page.logo-alt": "Logo",
+  "scene-page.made-with": "fait avec <a/>",
+  "scene-page.remix-attribution": "(Remixé depuis <a>{name} bpar {author}</a>)",
+  "scene-page.remix-button": "Remixé dans {editorName}",
+  "scene-page.scene-attribution": "par {creator}",
+  "scene-page.tweet-button": "Partager sur Twitter",
+  "scene-page.unavailable": "Cette scène n'est plus accessible.",
+  "scene-page.unknown": "inconnu",
+  "scene-url-modal.change-scene-button": "Changer la Scène",
+  "scene-url-modal.create-in-spoke": "Ou, créez une nouvelle scène en utilisant {editorName}.",
+  "scene-url-modal.message": "Coller une URL vers une scène ou une URL vers un <glblink>GLB</glblink>.",
+  "scene-url-modal.message-with-spoke": "Collez une URL vers une scène {editorName} ou une URL vers un <glblink>GLB</glblink>.",
+  "scene-url-modal.new-spoke-project-button": "Charger {editorName}",
+  "scene-url-modal.title": "Custom Scene URL",
+  "scene-url-modal.url-input": "Scene URL",
+  "select-input-field.placeholder": "Sélectionner...",
+  "share-popover.source.camera": "Camera",
+  "share-popover.source.screen": "Ecran",
+  "share-popover.title": "Partager",
+  "sign-in-modal.complete": "Vous êtes maintenant connecté(e).",
+  "sign-in-modal.newsletter-signup-link": "S'inscrire pour les mises à jour",
+  "sign-in-modal.newsletter-signup-question": "Voulez vous que les news de Hubs soient envoyées dans votre boîte de réception?",
+  "sign-in-modal.prompt": "Connectez vous s'il vous plait",
+  "sign-in-modal.signin-message.add-owner": "Vous devrez vous connecter pour assigner des modérateurs.",
+  "sign-in-modal.signin-message.change-scene": "Vous devrez vous connecter pour changer la scène.",
+  "sign-in-modal.signin-message.close-room": "Vous devrez vous connecter pour fermer la salle.",
+  "sign-in-modal.signin-message.create-avatar": "Vous devrez vous connecter pour créer des avatars.",
+  "sign-in-modal.signin-message.favorite-room": "Vous devrez vous connecter pour ajouter cette salle à vos favorites.",
+  "sign-in-modal.signin-message.favorite-rooms": "Vous devrez vous connecter pour ajouter des rooms favorites.",
+  "sign-in-modal.signin-message.kick-user": "Vous devrez vous connecter pour éjecter les autres utilisateurs.",
```

Comment:
```suggestion
  "sign-in-modal.signin-message.kick-user": "Vous devrez vous connecter pour bannir d'autres utilisateurs.",
```

Reply:
kick = éjecter
ban = bannir

Reply:
Yes you're right, I wasn't sure about "éjecter" and tried to find something like "expulser" but "éjecter" is more appropriate